pkg/agent/utils.go

```diff
@@ -808,3 +840,136 @@ func getCustomDataFromJSON(jsonStr string) string {
 	}
 	return customDataObj["customData"]
 }
+
+// GetOrderedKubeletConfigString returns an ordered string of key/val pairs
+// copied from AKS-Engine and filter out flags that already translated to config file
+func GetOrderedKubeletConfigString(k *api.KubernetesConfig, cs *api.ContainerService) string {
+	keys := []string{}
+	dynamicKubeletSupported := IsDynamicKubeletSupported(cs)
+	for key := range k.KubeletConfig {
+		if !dynamicKubeletSupported || !TranslatedKubeletConfigFlags[key] {
+			keys = append(keys, key)
+		}
+	}
+	sort.Strings(keys)
+	var buf bytes.Buffer
+	for _, key := range keys {
+		buf.WriteString(fmt.Sprintf("%s=%s ", key, k.KubeletConfig[key]))
+	}
+	return buf.String()
+}
+
+// IsDynamicKubeletSupported get if dynamic kubelet is supported in AKS
+func IsDynamicKubeletSupported(cs *api.ContainerService) bool {
+	return cs.Properties.OrchestratorProfile.IsKubernetes() && IsKubernetesVersionGe(cs.Properties.OrchestratorProfile.OrchestratorVersion, "1.14.0")
```

Comment:
I assume we only care about new create clusters so I put 1.14.0 here for lowest support version.

Reply:
can we get a flag from caller, like enableGPUDevicePlugin? We should be able to turn this off if something doesn't work as expected?

Reply:
Can we hardcode this to false and merge this PR first? 
There are more open PR to agentbaker and I want the unit test change to be in ASAP so we can ask them to add unit test for their change

Reply:
hard coded as false and merge this PR for now